### PR TITLE
Multiple reward tokens

### LINKED ISSUE #454: `IAsset.getClaimCalldata` does not support multiple claim steps
We should probably use the delegatecall pattern over the prepare-calldata pattern here, so that the collateral plugin can specify arbitrary logic and not be constrained to a single call. Alternatively we could change the function to key off the rewardToken and perform multiple claims.

### LINKED ISSUE #455: `IAsset.rewardToken` does not support multiple tokens, preventing sweeping
It could be an array so that sweeping can happen for all the reward tokens.

### PR DESCRIPTION
**Pull request by tbrent**
Resolves #454
Resolves #455

### CODE REVIEW

File: contracts/p1/mixins/RewardableLib.sol [hunk 1]
```diff
@@ -114,13 +72,34 @@ library RewardableLibP1 {
...
-                IERC20Upgradeable(address(erc20s[i])).safeTransfer(address(bm), deltas[i]);
-                require(
-                    IERC20Upgradeable(address(erc20s[i])).balanceOf(address(this)) >=
-                        liabilities[erc20s[i]],
-                    "missing liabilities"
+                IERC20(address(erc20s[i])).safeTransfer(address(bm), deltas[i]);
+
+                // Verify nothing has gone wrong -- we should keep this assert, even in P1
+                assert(
+                    IERC20(address(erc20s[i])).balanceOf(address(this)) >= liabilities[erc20s[i]]
```
[the remark this answers is not in the document]
Comment (fiddlemath) [reply contracts/p1/mixins/RewardableLib.sol:79]: Pretty sure this should just work? But that's a _normative_ should, not an _epistemic_ one.

File: contracts/plugins/assets/ATokenFiatCollateral.sol [hunk 2]
```diff
@@ -2,32 +2,30 @@
...
 import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
-import "@openzeppelin/contracts/utils/math/Math.sol";
 import "contracts/libraries/Fixed.sol";
 import "contracts/plugins/assets/AbstractCollateral.sol";
 
-// ==== External ====
-
-// External interfaces from: https://git.io/JX7iJ
+// This interface is redundant with contracts/plugins/aave/IStaticAToken, but it's compiled with
+// a different version
```
Comment (fiddlemath) [inline contracts/plugins/assets/ATokenFiatCollateral.sol:9]: hehe -- I claim that you don't really need to apologize for abstracting a contract with an interface. :)

**Review by fiddlemath — APPROVED** [review]
This looks great! There's a few purely stylistic tweaks I'd recommend, but nothing that matters enough to prevent pushing the branch.

File: contracts/p1/mixins/RewardableLib.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (tbrent) [reply contracts/p1/mixins/RewardableLib.sol:79]: Yep it should, that was a bit silly

File: contracts/plugins/assets/ATokenFiatCollateral.sol (hunk 2 above)
Comment (tbrent) [reply contracts/plugins/assets/ATokenFiatCollateral.sol:9]: The apology is for duplicate interfaces in the same repo, for the same contract. Leaving a slightly tweaked version of the note

**Pull request merged**